### DISCUSSION TITLE
Error starting lotus "ERROR: failed to start json-rpc endpoint: could not listen: listen tcp4 127.0.0.1:1234 bind: address already in use"

### DISCUSSION THREAD

**Discussion by theasianbull** [filecoin-project/lotus#7472] [General Q&A] [answered]
Please I need your help. This is how i set up.
1st drive 1 tb is boot, it have the lotus clone from github.
2nd drive: /.lotus ( 1tb nvme)
3rd drive: /Lotus (1 tb nvme)
4 drive: /proof and 256 Gb swap ( 1tb nvme)
5 drive :/backup ( 1 tb nvme)
6 drive :/storage (14 gb hdd)

before i run the lotus daemon for the first time i apply my env
export LOTUS_MINER_PATH=/storage
export LOTUS_WORKER_PATH=/Lotus/worker
export TMPDIR=/Lotus/tmpdir
export FIL_PROOFS_PARAMETER_CACHE=/proof
export FIL_PROOFS_PARENT_CACHE=/home/proof
export LOTUS_PATH=/.lotus
export LOTUS_BACKUP_BASE_PATH=/backup
export FIL_PROOFS_USE_GPU_COLUMN_BUILDER=1
export FIL_PROOFS_USE_GPU_TREE_BUILDER=1
export FIL_PROOFS_SDR_PARENTS_CACHE_SIZE=1073741824
export RUST_BACKTRACE=full
export RUST_LOG=info
export RUSTFLAGS="-C target-cpu=native -g"
export FFI_BUILD_FROM_SOURCE=1
export FIL_PROOFS_MAXIMIZE_CACHING=1
export FIL_PROOFS_USE_MULTICORE_SDR=1
export LOTUS_MAX_HEAP=27002396672
ulimit -n 1048576

after download the proof my lotus run an error:

ERROR: failed to start json-rpc endpoint: could not listen: listen tcp4 127.0.0.1🔢 bind: address already in use

when run lotus sync wait
ERROR: could not get API info for FullNode: could not get api endpoint: API not running (no endpoint)

**Comment by jburnett** [comment, marked as answer]
As the error indicates, lotus is attempting to share its API over a port which is already being used. Check the API.ListenAddress setting in /Lotus/config.toml and determine what is using that port.  Change the ListenAddress to an unused port.

**Reply by LexLuthr** [reply]
@theasianbull Hey, do you need any further help with this problem? Were you able to identify what was using the default lotus port and fix it?

**Reply by theasianbull** [reply]
I didnt know what wrong, i reinstall everything and it fix the problem. Appreiciate that you check on me!
